### PR TITLE
Feat: Added input section to library

### PR DESCRIPTION
**Pull request by abhijeetekad**
Added Input sections to the library:
1)Input- Text area.
2)Input's with different labels.
3)Errored / successful inputs.
4)Inputs with options.
CPT2202042149-1521x714

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]
Good to merge!

File: components/input.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+}
+.input-section h2 {
+  font-size: 48px;
+  margin: 2rem 0rem 2rem 0rem;
+}
+.input-section h4 {
+  font-size: 24px;
```
Comment (Manpreet-hub) [inline components/input.css]: Suggestion: don't mix rem/px unit. Try to use rem unit.

**Review by Manpreet-hub — CHANGES_REQUESTED** [review]
Added one suggestion.

File: components/input.css (hunk 1 above)
Comment (abhijeetekad) [reply components/input.css]: Thanks for the suggestion. I have changed that.

**Pull request merged**